### PR TITLE
Replace boilerplate fork repo with upstream repo

### PR DESCRIPTION
**Pull request by jeff-matthews**
This pull request (PR) replaces the boilerplate repo URL in the [getting started tutorial](https://experienceleague.adobe.com/developer/commerce/storefront/get-started/) with the new URL. 

The team recently announced a switch from the fork (https://github.com/hlxsites/boilerplate-commerce-dropins) to the upstream repo (https://github.com/hlxsites/aem-boilerplate-commerce).

I validated that these instructions are still accurate and successfully launched a new storefront (https://github.com/jeff-matthews/storefront-tutorial) based on the `main` (default) branch in the upstream repo.

## `develop` branch

The default branch in the upstream repo is `main` and this tutorial instructs users to **not include all branches** during set up.

Dropins that are in development are available in the `develop` branch. We _could_ add a note that if users want to test unreleased dropins, then they can add the `develop` branch to their repo using `git` commands. However, I think it's better to stick with `main` for the tutorial and not add confusion about `develop` for now. We should start positioning our docs for GA.

>[!NOTE]
>Unrelated, but I also updated the demo store URL per PM request since I was touching this file.

### CODE REVIEW

File: src/content/docs/get-started/index.mdx [hunk 1]
```diff
@@ -83,9 +83,9 @@ The `mountpoint` is the URL that points to your content folder on Google Drive o
 
 ## Example
 
-Our WKND demo site was built from the same boilerplate you will set up to develop your own storefront. You can access the full WKND demo site here: https://main--xsc-wknd-commerce--hlxsites.hlx.live/.
+Our CitiSignal demo site was built from the same boilerplate you will set up to develop your own storefront. You can access the full demo site here: https://main--citisignal--adobedevxsc.hlx.page/.
 
-<Embed src="https://main--xsc-wknd-commerce--hlxsites.hlx.live/" height="520" />
+<Embed src="https://main--citisignal--adobedevxsc.hlx.page/" height="520" />
```
Comment (sirugh) [inline src/content/docs/get-started/index.mdx]: Did you mean to use `.page`? Also I think we can and should start using `aem.live` or `aem.page`.

cc @herzog31

File: src/content/docs/get-started/index.mdx [hunk 2]
```diff
@@ -178,10 +178,10 @@ Make sure you are signed into your GitHub account before you start. Otherwise, y
 
 <Callouts columnCount="1">
 
-1. Navigate to <Link href="https://github.com/hlxsites/boilerplate-commerce-dropins" text="boilerplate-commerce-dropins" />, select the **Use this template** button, then select the **Create a new repository** option to open the form.
+1. Navigate to <Link href="https://github.com/hlxsites/aem-boilerplate-commerce" text="aem-boilerplate-commerce" />, select the **Use this template** button, then select the **Create a new repository** option to open the form.
 1. Complete the form with the following details:
 
-   - **Repository template**: `hlxsites/boilerplate-commerce-dropins` (default).
+   - **Repository template**: `hlxsites/aem-boilerplate-commerce` (default).
    - **Include all branches**: Do not include all branches (default).
```
Comment (sirugh) [inline src/content/docs/get-started/index.mdx:185]: This is fine, and I think we can potentially include some manual instructions for checking the specific `develop` branch from the boilerplate, if desired.

File: src/content/docs/get-started/index.mdx (hunk 1 above)
Comment (jeff-matthews) [reply src/content/docs/get-started/index.mdx]: I actually don't know the difference between the two 😅. I just used the link that Olena sent me. 

Is this the preferred link?

https://main--citisignal--adobedevxsc.aem.live/

File: src/content/docs/get-started/index.mdx (hunk 2 above)
Comment (jeff-matthews) [reply src/content/docs/get-started/index.mdx:185]: Yeah, now that I think of it, we could probably add steps to the [local development](https://experienceleague.adobe.com/developer/commerce/storefront/get-started/#set-up-local-dev-environment) section at the end of the tutorial (or a new section just below that after they've successfully launched locally).

File: src/content/docs/get-started/index.mdx (hunk 1 above)
Comment (sirugh) [reply src/content/docs/get-started/index.mdx]: IIUC there are caching differences, but also there can be differences such as which backend is used.

We map .live to production environment, .page to staging, and localhost to integration cloud environments, so depending on which url you use, you may get different reuslts. I would double check with Olena or even the sales demo team working on this, which URL they would like to have. I added you to the internal slack channel.

**Review by sirugh — APPROVED** [review]

**Review by bdenham — APPROVED** [review]

File: src/content/docs/get-started/index.mdx (hunk 1 above)
Comment (jeff-matthews) [reply src/content/docs/get-started/index.mdx]: Confirmed with Olena that it should be `https://main--citisignal--adobedevxsc.aem.live/`.

**Pull request merged**